elements/pfe-switch/src/pfe-switch.scss

```diff
@@ -0,0 +1,219 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: switch;
+
+// PFE Config
+$font-size:     16px;
+$line-height:   24px;
+
+// Setting all toggle sizing, will be converted to em in styles
+$toggle-width:        42px;
+$toggle-height:       $line-height;
+$toggle-knob-diameter:   14px;
+$toggle-knob-offset:  5px;
+
+///
+/// Convert px to em
+///
+/// @param $value {length}  Value to convert with px suffix
+/// @param $base-size {length} Optional, font size to use to calculate em
+/// @return {length} Em version of input
+@function px-to-em($value) {
+  @return $value / $font-size * 1em;
+}
+
+//sass-lint:disable indentation
+// Variables based on PF Next Switch
+$variables: (
```

Comment:
This is looking really great!  We don't have any linting on this project so the sass-lint comment is not necessary.

Reply:
Sorry, I have it locally and I didn't like seeing the 'error' underline 😛 will be removed!